---
color_correction:
  percentile: 5
  lamda: 0.2

biletral_filter:
  sigma_f: 0.1
  sigma_g: 0.1

rghs:
  percentile: 0.1
  distance: 3

fusion:
  pyr_level: 6
  exposedness_sigma: 0.25
